homeassistant/components/yeelight/light.py

```diff
@@ -376,6 +389,10 @@ def _brightness_property(self):
     def _power_property(self):
         return "power"
 
+    @property
+    def _turn_on_power_mode(self):
+        return PowerMode.LAST
```

Comment:
Does this change the previous behavior?

Reply:
It keeps default behavio. By default power mode is set to PowerMode.LAST , in turn_on call ( https://gitlab.com/stavros/python-yeelight/blob/master/yeelight/main.py#L195 and https://gitlab.com/stavros/python-yeelight/blob/master/yeelight/main.py#L52 ). It states that explicitly

Reply:
Ok, let's get this merged, thanks for your patience :-)